modules/SequencesExt.tla

```diff
@@ -18,19 +18,6 @@ ToSet(s) ==
   (*************************************************************************)
   { s[i] : i \in DOMAIN s }
 
-IsInjective(s) == 
```

Comment:
We will have to update the Java module to now bind it to `Functions!IsInjective`.  For as long as we don't have a generic module override that works for all functions, the existing one can just return `null` for non-tuples, which causes TLC to fall back to the TLA+ definition.

Reply:
Hmm, not sure what to do here. I didn't mean to change SequencesExt.tla, although I may have messed up since I started from a stale branch of CommunityModules and only realized this later. Also, I don't see any binding for this operator in modules/tlc2/overrides/SequencesExt.java. May I ask you to do whatever is appropriate? Thanks.

Reply:
Not sure what happened here, but I now squashed your commits and pushed `BagsExt` and its auxiliary files to master in https://github.com/tlaplus/CommunityModules/commit/fbe0f276172eeb50853e44ad4970bbb535e72511